features/migrate_client_to_minion.feature

```diff
@@ -25,3 +28,50 @@ Feature: Migrate a traditional client into a salt minion
      # bsc#1020902 - moving from traditional to salt with bootstrap is not disabling rhnsd
      When I run "systemctl status nhsd" on "sle-migrated-minion"
      Then the command should fail
+
+  Scenario: Check that the migrated system is now a minion
+     # bsc1023399 - Migrate a traditional client in SSH push mode into a salt minion doesn't work
+     Given I am on the Systems overview page of this "sle-migrated-minion"
+     When I follow "Properties" in the content area
+     Then I should see a "Base System Type:     Salt" text
+
+  Scenario: Check that channels are still the same after migration
+     Given I am on the Systems overview page of this "sle-migrated-minion"
+     When I follow "Software" in the content area
+     And I follow "Software Channels" in the content area
+     Then I should see a "SLES11-SP3-Updates x86_64 Channel" text
+     And I should see a "Test Base Channel" text
+
+  Scenario: Check that groups are still the same after migration
+     Given I am on the Systems overview page of this "sle-migrated-minion"
+     When I follow "Groups" in the content area
+     Then I should see a "newgroup" text
+
+  Scenario: Check that events history is still the same after migration
+     Given I am on the Systems overview page of this "sle-migrated-minion"
+     When I follow "Events" in the content area
+     And I follow "History" in the content area
+     Then I should see a "Subscription via Token" text
+
+  Scenario: Install a package onto the migrated minion
+     Given I am on the Systems overview page of this "sle-migrated-minion"
+     When I follow "Software" in the content area
+     And I follow "Install"
+     And I check "perseus-dummy-1.1-1.1" in the list
+     And I click on "Install Selected Packages"
+     And I click on "Confirm"
+     And I wait for "5" seconds
+     Then I should see a "1 package install has been scheduled for" text
+     And I wait for "perseus-dummy-1.1-1.1" to be installed
+
+  Scenario: Run a remote command on the migrated minion
+     Given I am on the Systems overview page of this "sle-migrated-minion"
+     When I follow "Remote Command" in the content area
+     And I enter as remote command this script in
+      """
+      #!/bin/bash
+      touch /tmp/tastes-better-with-salt
+      """
+     And I click on "Schedule"
+     Then I should see a "Remote Command has been scheduled successfully" text
+     And "/tmp/tastes-better-with-salt" exists on the filesystem
```

Comment:
I saw the implementation here ```features/step_definitions/salt_steps.rb``, @ebischoff, if you want to eliminate the step and make it better, for me is ok. otherwise keep the old one just because it works, and use a new correct one, as you wish